client/routes/home.js

```diff
@@ -7,11 +7,17 @@ module.exports = function (page) {
   page('/', render, listeners);
 
   function render (ctx, next) {
-    var pageEl = document.getElementsByTagName('main')[0];
-    pageEl.innerHTML = Templates['home'](data);
-    // refresh the title of the page
-    document.getElementsByTagName('title')[0].textContent = data.titlePage;
-    next();
+    if (!ctx.isInitialRender) {
+      var pageEl = document.getElementsByTagName('main')[0];
+      data.footer = require('../../fixtures/footer');
+      data.footerBanner = require('../../fixtures/footer-banner');
+      pageEl.innerHTML = Templates['home'](data);
+      // refresh the title of the page
+      document.getElementsByTagName('title')[0].textContent = data.titlePage;
+      next();
+    } else {
+      listeners();
```

Comment:
This should be a call to next also, so we don't introduce a bug in the future when the `listeners` function also needs to use `ctx`.